client/src/components/ReviewListEntry.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from 'react';
+
+class ReviewListEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  componentDidMount() {
+  }
+
+  // HTTP Request Handlers
+  // Set State
+  // Event Handlers
+
+  render() {
+    return (
+      <Fragment>
+        <div>★★★★☆ April 30, 2020</div>
```

Comment:
I can't wait to see how you implement these stars! *eyes emoji*

Reply:
Haha. You'll have to implement it too!